pkg/envtest/server.go

```diff
@@ -34,6 +34,17 @@ const (
 	StopTimeout             = 60
 )
 
+// APIServerDefaultArgs are flags necessary to bring up apiserver.
+// TODO: create test framework interface to append flag to default flags.
+var defaultKubeAPIServerFlags = []string{
+	"--etcd-servers={{ if .EtcdURL }}{{ .EtcdURL.String }}{{ end }}",
+	"--cert-dir={{ .CertDir }}",
+	"--insecure-port={{ if .URL }}{{ .URL.Port }}{{ end }}",
+	"--insecure-bind-address={{ if .URL }}{{ .URL.Hostname }}{{ end }}",
+	"--secure-port=0",
+	"--admission-control=AlwaysAdmit",
+}
```

Comment:
Didn't see this in any other places where we were using testing framework. Will dig more into it.

Reply:
This adds the "--admission-control=AlwaysAdmit" flag to the other default flags (unfortunately it's not possible to append a flag to the defaults) for the kube apiserver. Arguably, this flag should be one of the defaults. Without it, it's not possible to add test Pods without running into the following types of errors:

`pods "test-pod-1" is forbidden: error looking up service account test-namespace-1/default: serviceaccount "default" not found`

We don't see this in other tests, because we either add Deployments or Services.

Reply:
Got it. Thanks for clarifying it.